blockpage.js

```diff
@@ -36,7 +36,7 @@ async function retryOnTabUpdate(tabId, info, tab) {
   }
 }
 
-chrome.runtime.onConnect.addListener((port) => {
+browser.runtime.onConnect.addListener((port) => {
```

Comment:
This all relies on chromes internal API for browser extensions and thus becomes disfunctional with these changes